src/sentry/api/endpoints/source_map_debug_blue_thunder_edition.py

```diff
@@ -416,7 +416,7 @@ def _find_source_file_in_artifact_indexes(self):
             self._get_dist_matched_artifact_index_release_file()
         )
         if dist_matched_artifact_index_release_file is not None:
-            raw_data = json.load(dist_matched_artifact_index_release_file.file.getfile())
+            raw_data = orjson.loads(dist_matched_artifact_index_release_file.file.getfile().read())
```

Comment:
oh interesting, this might be the leaked socket I've been looking for

each call to `getfile` should be wrapped in context manager otherwise it leaks a file descriptor

no need to fix here though

Reply:
nice